docs/content/Header.mdx

```diff
@@ -54,25 +54,25 @@ All items directly under the Header component should be a `Header.Item` componen
 </Header>
 ```
 
-## Component props
+## Props
 
-### Header
+### `Header`
 
-| Name | Type              | Default | Description                          |
-| :--- | :---------------- | :-----: | :----------------------------------- |
-| sx   | SystemStyleObject |   {}    | Style to be applied to the component |
+<PropsTable>
+  <PropsTableSxRow />
+</PropsTable>
 
-### Header.Item
+### `Header.Item`
```

Comment:
```suggestion
### Header.Item
```

Reply:
Updated. But I do think the syntax-like content makes this quite an ugly heading